flixel/addons/editors/tiled/TiledTileSet.hx

```diff
@@ -58,12 +59,16 @@ class TiledTileSet
 		
 		if (source.has.source)
 		{
-			var sourcePath = rootPath + source.att.source;
+			var sourcePath = haxe.io.Path.normalize(rootPath + source.att.source);
```

Comment:
No need to fully qualify the path with `haxe.io` if you already imported the class. ;)